vocabularies/qg-agents.ttl

```diff
@@ -0,0 +1,102 @@
+@prefix qgorg: <https://linked.data.gov.au/qg-agent/> .
+@prefix qgsite: <https://linked.data.gov.au/qg-sites/> .
+@prefix org: <http://www.w3.org/ns/org#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dcat: <http://www.w3.org/ns/dcat#> .
+@prefix prov: <http://www.w3.org/ns/prov#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vcard: <http://www.w3.org/2006/vcard/ns#> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+ 
+@base <https://linked.data.gov.au/qg-agent> .
+
+<https://linked.data.gov.au/qg-agent> rdf:type owl:Ontology , skos:ConceptScheme ;
+      owl:imports <http://www.w3.org/2006/vcard/ns> ,
+         org: ,
+         <http://www.w3.org/ns/prov-o#> ;
+      dcterms:created "2020-05-21" ;
+      dcterms:creator <https://orcid.org/0000-0002-3884-3420> ,
+                      <https://orcid.org/0000-0003-0395-9703> ,
+                      <http://linked.data.gov.au/org/des> ;
+      dcterms:publisher <http://linked.data.gov.au/org/gsq> ;
+      dcterms:modified "2020-12-01" ;
+      dcterms:title "Queensland Governement organizations and agencies"@en ;
+      skos:prefLabel "QG Organisations"@en ;
+      rdfs:comment "RDF graph of Queensland Government organisations (departments) which may include some of the departmental divisions, used as an initial starting point for the DES Data Catalogue. Not a full organisational hierarchy but limited to higher levels which are useful to identify."@en .
+
+
+###  https://linked.data.gov.au/qg-agent/DES
+qgorg:DES rdf:type owl:NamedIndividual ,
+              org:FormalOrganization ,
+              prov:Organization ,
+              skos:Concept ;
+     org:hasPrimarySite qgsite:GeorgeSt ;
+     org:hasSite qgsite:Ecosciences ;
+     rdfs:label "Department of Environment and Science"@en ;

```

Comment:
```suggestion

```

Reply:
Please remove rdfs:label throughout the document. It is duplicative of SKOS:preflabel.